tests/typings/tsd.d.ts

```diff
@@ -1,3 +1,6 @@
 /// <reference path="intern/intern.d.ts" />
 /// <reference path="benchmark/benchmark.d.ts" />
 /// <reference path="sinon/sinon.d.ts" />
+/// <reference path="formidable/formidable.d.ts" />
+/// <reference path="http-proxy/http-proxy.d.ts" />
+/// <reference path="services/echo.d.ts" />
```

Comment:
Are we still doing typings in a tsd, or should they just be defined in the `tsconfig.json`?